praw/objects.py

```diff
@@ -636,6 +636,33 @@ class Message(Inboxable):
 
     """A class for private messages."""
 
+    @staticmethod
+    @restrict_access(scope='privatemessages')
+    def from_id(reddit_session, message_id, *args, **kwargs):
+        """Request the url for a Message and return a Message object.
+
+        :param reddit_session: The session to make the request with.
+        :param message_id: The ID of the message to request.
+
+        The additional parameters are passed directly into
+        :meth:`.request_json`.
+
+        """
+        # Reduce fullname to ID if necessary
+        message_id = message_id.replace('t4_', '')
```

Comment:
How would you like this to look? At first I thought:

    if '_' in message_id:
        message_id.split('_')[1]

but by the same token I guess I shouldn't assume the separator will always be an underscore. The method doesn't necessarily need to support fullnames, handling that can be the user's responsibility.